### PR TITLE
COMMERCE-619

### PR DESCRIPTION
**Pull request by marco-leo**
https://issues.liferay.com/browse/COMMERCE-619

As a catalog manager I want to enable subscription on a Product

//cc @Commando18 @luca-pellizzon

### CODE REVIEW

**Comment by marco-leo** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

File: commerce-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -209,6 +209,8 @@ manufacturer-part-number=Manufacturer Part Number
 manufacturer-part-number-x=Manufacturer Part Number: {0}
 mappings=Mappings
 max-order-quantity=Maximum Order Quantity
+max-subscription-cycles-number=Max Subscriptions Cycles
```
Comment (ethanbustad) [inline commerce-lang/src/main/resources/content/Language.properties:212]: Should be `Max Subscription Cycles` rather than `Max Subscriptions Cycles`.

**Review by ethanbustad — CHANGES_REQUESTED** [review]
Other than that one language property, looks really great!

**Pull request closed without merging**